.gitlab/deps_build.yml

```diff
@@ -83,4 +83,17 @@ build_omnibus-nikos_arm64:
   tags: ["runner:docker-arm", "platform:arm64"]
   variables:
     ARCH: arm64
-  
\ No newline at end of file
+
+build_vckpg_deps:
```

Comment:
We could go without this job and `tests_windows_base` would create the cache anyway, am I right? If that's the case maybe we don't need to add this one as a separate job.